debezium-testing/debezium-testing-testcontainers/pom.xml

```diff
@@ -41,6 +41,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>postgresql</artifactId>
```

Comment:
Not sure, but maybe this could be `<scope>provided</scope>` , but I'm not sure about it. Also debezium-testing-testcontainers is only a `test` scope dependency it should not impact production JARs negatively, wdyt?

Reply:
Because it is used in `./src/` now.